jolly-jellyfish/src/page_maker/tests.py

```diff
@@ -38,13 +50,115 @@ def test_register_and_home(self):
         self.assertNotIn(USERNAME.encode(), res, 'Our username was already in the home view.')
 
 
+class UserViewsTest(TestCase):
+    fixtures = ['users.yaml', 'webpages.yaml', 'templates.yaml']
+
+    def test_user_detail(self):
+        user = User.objects.get(pk=1)
+        res = self.client.get('/users/1/')
+        self.assertEqual(res.status_code, 200)
+        self.assertEqual(res.context['viewed_user'], user)
+        self.assertIsInstance(res.context['user_pages'][0], Webpage)
+
+    def test_user_update(self):
+        self.client.login(username=USERNAME, password=PASSWORD)
+        res = self.client.post('/users/2/update', {
+            'first_name': 'john', 'last_name': 'lemon', 'email': 'jl@sth.com'
+        })
+        self.assertRedirects(res, '/')
+        user = User.objects.get(pk=2)
+        self.assertEqual(user.last_name, 'lemon')
+
+    def test_user_delete(self):
+        self.client.login(username=USERNAME, password=PASSWORD)
```

Comment:
We could probably move this line to the `setUp`, given that `test_user_detail` shouldn't be dependent on a user being signed in.